moped-database/migrations/1643996372208_moped_proj_features_changes/up.sql

```diff
@@ -0,0 +1,13 @@
+
+ALTER TABLE "public"."moped_proj_features" ADD COLUMN "project_component_id" integer NULL;
+
+alter table "public"."moped_proj_features"
+           add constraint "moped_proj_features_project_component_id_fkey"
+           foreign key ("project_component_id")
+           references "public"."moped_proj_components"
+           ("project_component_id") on update cascade on delete cascade;
+
+-- this would ideal but will break in staging or prod
+-- ALTER TABLE "public"."moped_proj_features" ALTER COLUMN "project_component_id" SET NOT NULL;
+
+alter table "public"."moped_proj_features" rename column "location" to "feature";
```

Comment:
> rename column "location" to "feature"

i think the name `feature` is clearer than `location`: this `JSONB` column is a single geojson [Feature](https://datatracker.ietf.org/doc/html/rfc7946#section-3.2) object. As in the geojson spec, we roll these features into a `FeatureCollection` in order to render them together on maps.

also considered `feature_geojson` to be even more explicit. i've definitely open to alternative names.